src/resources/users/user-controllers.js

```diff
@@ -25,13 +25,15 @@ exports.accountRecovery = async (req, res) => {
     }
   } catch (error) {
     res.status(401).json({
-      message: 'password reset link is invalid or has expired',
+      error: {
+        message: 'password reset link is invalid or has expired',
+      },
     });
   }
 };
 
 exports.resetPasswordEmail = async (req, res) => {
-  const { email } = req.body;
+  const email = req.body.email;
```

Comment:
why?

Reply:
No particular reason actually ...more readability for someone without destructuring knowledge I guess

Reply:
If there's someone on the team that does not understand the basics of ES2015, I'd rahter skill that person up than to refactor code

Reply:
Touche